src/components/Session/Session.tsx

```diff
@@ -2,7 +2,7 @@ import { ApolloCache, useQuery } from '@apollo/client';
 import { pickBy } from 'lodash';
 import LogRocket from 'logrocket';
 import { useEffect } from 'react';
-import { SessionOutput } from '~/api/schema.graphql';
+import { BetaFeatures, SessionOutput } from '~/api/schema.graphql';
```

Comment:
Can you use the type from the fragment you added instead of this schema type? It'll prevent checking for a feature that hasn't been fetched in future.